ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/minimal-permission/byo-hosted-zone/private/provision/cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone-private-provision-chain.yaml

```diff
@@ -0,0 +1,44 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone-private-provision
+  steps: 
+    - ref: upi-conf-gcp-xpn
+    - ref: ignition-bastionhost
+    - ref: gcp-provision-bastionhost
+    - ref: proxy-config-generate
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-credentials-mode
+    - ref: ipi-conf-gcp-xpn-network
+    - ref: ipi-conf-gcp-create-firewall-rules
+    - ref: ipi-conf-gcp-basedomain
+    - ref: gcp-provision-private-dns-zone
+    - ref: gcp-provision-xpn-byo-hosted-zone-min-perm-passthrough-serviceaccount
+    - ref: ipi-conf-gcp-service-account
+    - ref: ipi-conf-gcp-private-dns-zone
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
+    - chain: cucushift-installer-check
+    - ref: cucushift-installer-check-gcp-private
+    - ref: cucushift-installer-check-gcp-credentials-mode
```

Comment:
considering we have dedicated step `cucushift-installer-check-gcp-byo-hosted-zone-post-cluster-destroy` to check the cluster's DNS private zone post the uninstallation, do we actually also need a check on the cluster's DNS private zone is created correctly or not after installation?

Reply:
FYI I'd ever tried post-installation check, but after discussing with Jianlin in https://github.com/openshift/release/pull/66951, we agreed that such check is not necessary, because if the cluster's DNS private zone has any issue the installation won't succeed. Does it make sense? @gpei